src/crate/sphinx/csv.py

```diff
@@ -71,10 +74,17 @@ def _apply_filter(self, rows, max_cols, exclude_dict):
         return result
 
     def _get_rows_with_included_cols(self, rows, included_cols_list):
-        my_rows = []
+        prepared_rows = []
         for row in rows:
-            my_rows.append([row[i] for i in included_cols_list])
-        return my_rows, len(included_cols_list)
+            try:
+                idx_row = [row[i] for i in included_cols_list]
+                prepared_rows.append(idx_row)
+            except IndexError:
+                error = self.state_machine.reporter.error(
+                        'The defined column indizes are insufficient. It is not '
```

Comment:
suggestion: One or more indexes of included_cols are not valid. The csv data does not contain that many columns.